bat/RUNASMDEMOS.BAT

```diff
@@ -11,46 +11,53 @@ if /I "%1" == "tron" (set z_TraceMode=tron
 set /A z_NestLevel=%z_NestLevel%+1
 rem ----- Lvl(%z_NestLevel%) Start %0 %1 %2 %3 %4 %5 %6 %7 %8 %9
 
+rem during build on github server the compare steps fail - no clue why.
+rem parm *Nocompare was introduced to enable us to suppress the compare steps during build
+rem but keep the default to do run the compare steps when the RUNASMDEMOS is invoked without the parm
+if /I %1. == *Nocompare. (set compflag=N
+                          shift /1
+                  ) else (set compflag=Y)
+
 pushd %~dps0..
 call bat\ASMLG %z_TraceMode% demo\HELLO    %1 %2 %3 %4 %5 %6 %7 %8 %9 || goto error
 call bat\ASM   %z_TraceMode% demo\DEMOM8Q1 %1 %2 %3 %4 %5 %6 %7 %8 %9 || goto error
+call bat\ASMLG %z_TraceMode% demo\DEMOBMK1 %1 %2 %3 %4 %5 %6 %7 %8 %9 || goto error
+call bat\ASMLG %z_TraceMode% demo\DEMOBMK2 %1 %2 %3 %4 %5 %6 %7 %8 %9 || goto error
+call bat\ASMLG %z_TraceMode% demo\DEMODFP1 %1 %2 %3 %4 %5 %6 %7 %8 %9 || goto error
+call bat\ASM   %z_TraceMode% demo\DEMONUM1 %1 %2 %3 %4 %5 %6 %7 %8 %9 || goto error
+call bat\ASMLG %z_TraceMode% demo\DEMONUM2 %1 %2 %3 %4 %5 %6 %7 %8 %9 || goto error
+call bat\ASMLG %z_TraceMode% demo\DEMOSTR1 %1 %2 %3 %4 %5 %6 %7 %8 %9 || goto error
+call bat\ASMLG %z_TraceMode% demo\DEMOWTO2 %1 %2 %3 %4 %5 %6 %7 %8 %9 || goto error
+call bat\ASMLG %z_TraceMode% demo\SIEVE    %1 %2 %3 %4 %5 %6 %7 %8 %9 || goto error
+call bat\ASMLG %z_TraceMode% demo\STDDEVLB %1 %2 %3 %4 %5 %6 %7 %8 %9 || goto error
+
 set sysut1=demo\TESTDCB1.TF1
 set sysut2=demo\TESTDCB1.TF2
 set sysout=demo\TESTDCB1.TF3
-call bat\ASMLG %z_TraceMode% demo\TESTDCB1 %1 %2 %3 %4 %5 %6 %7 %8 %9 || goto error
-::comp /M demo\TESTDCB1.TF1 demo\TESTDCB1.TF2
-::set z_ReturnCode=%ERRORLEVEL%
-::if %z_ReturnCode% GTR 0 (set z_ReturnCode=8
-::                         echo %0 ERROR: TESTDCB1 data were not copied correctly
-::                         goto return
-::
-::                         )
-:: next 4 lines added to temporarily match bash version
-:: type demo\TESTDCB1.TF2
-:: echo "Verify demo\TESTDCB1 copied demo\TESTDCB1.TF1 to demo\TESTDCB1.TF2"
-:: echo "Verify demo\TESTDCB1 wrote to demo\TESTDCB1.TF3"
+                      call bat\ASMLG %z_TraceMode% demo\TESTDCB1 %1 %2 %3 %4 %5 %6 %7 %8 %9 || goto error
```

Comment:
Line appears to be indented???

Reply:
I aligned with the call statement on the next line - they belong together as a pair.